### ISSUE TITLE
[Doc] Need updated documentation format/content for dropdown (+ nested menu)

### ISSUE THREAD

**Issue by jeeyun** [vmware/clarity#871]
We need to update the dropdown documentation to follow our new design.

For reference, these are already updated to newer format/content:
https://vmware.github.io/clarity/documentation/buttons
https://vmware.github.io/clarity/documentation/modals

Once I get the content/images from @lil-kim I can work on the actual updating the website or coordinate with her.

**Comment by lil-kim** [comment]
### Plz refer to Sketch file for layout:
[Nested menu doc.zip](https://github.com/vmware/clarity/files/1005448/Nested.menu.doc.zip)

------------------------------ _Begin Doc_ -----------------------------
(h1) Nested Menus 

(h5) Nested menus save space by organizing options or actions into categories. The user can click on a group name to reveal deeper levels.

(h3) Description	

(p1)
- Same visual style as standard dropdown menu with the exception of a darker border (#9A9A9A)
- Slight overlap (4px) over root menu to emphasize stack order
- Width: Minimum 168 px; Maximum 336 px
- Default ine height is 30 px; 36 px for x-small breakpoint (<544px)

(h3) Interaction

(p1)
- With mouse: Click group name to reveal nested menus. Group name stays in selected state to serve as a breadcrumb 
- Clicking outside of the menu dismisses all menus

![img_nested levels](https://cloud.githubusercontent.com/assets/23196444/26121434/0b6734da-3a29-11e7-9079-63c4e77d1b30.png)

(h3) Placement

_(column 1-Left)_
![img_placement_default](https://cloud.githubusercontent.com/assets/23196444/26121474/3aa2cdf4-3a29-11e7-866c-bcb6f545a06a.png)

(h6) Default placement and stack order
(p1) By default a nested menu pops up on the right side of its root menu.

_(column 2-right)_
![img_placement_resp](https://cloud.githubusercontent.com/assets/23196444/26121489/4a155356-3a29-11e7-8cd8-98bea6d4d84d.png)

(h6) Small screen / responsive placement and stack order (<544px) 
(p1) If screen space isn’t available to the right of the root menu, the nested menu appears on the left side. If there is a second nested menu, it appears to the right of the first nested menu, creating a zig-zag pattern.
------------------------------ _End of Doc_ -----------------------------

**Comment by lil-kim** [comment]
**Images for Doc**
[images_png.zip](https://github.com/vmware/clarity/files/1005465/images_png.zip)

**Comment by lil-kim** [comment]
@jeeyun Thanks for getting this done, it looks great! 

I only see a couple of small items.
1) Text should be updated in the Dropdown vs Select box panel:

Should be:

(h6) Dropdown menus  (p1) present a list of options that take an immediate action or navigate the user outside of the current context.

(h6) Select Boxes (p1) present a list of options that the user can choose from and apply to an input field.

2) Is it possible to move the purple circle w/ "VS" more to the center of the panel?

3) Padding issue:
Need 24px space between h3 titles and following bulleted lists. Or follow what is being used in the button doc. 
--------------
Thank you!

**Comment by jeeyun** [comment]
Hi @lil-kim I created a separate PR to address your comments. Unfortunately I couldn't find a cross-browser compatible way to address your point 2 about the "VS". I'll leave that as is for now, but the rest should be updated once my PR is merged.
